packages/ember-principled-forms/jsconfig.json

```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"target":"es6","experimentalDecorators":true},"exclude":["node_modules","bower_components","tmp","vendor",".git","dist"]}
```

Comment:
**?**: This doesn't appear to be relevant to the commit. Why was it included?

Reply:
Heh, this is autogenerated stuff from VS Code, I think possibly turned on by one of the packages in the Ember VS Code collection. I've never figured out how to keep it from being generated, and now it bit @billpull as well.

Reply:
Ah, I see.